lib/plugins/loop.js

```diff
@@ -231,7 +231,7 @@ function init (ctx) {
         , split: false
         , targets: false
         , reasons: reasonconf
-        , otp: true 
+        , otp: false //TODO: was why was this set to true? I don't think we need that, maybe an option?
```

Comment:
The intention is for loop to have the option to require an OTP even for remote overrides. It was set to true on purpose. The current loop code ignores it for now, but this makes the OTP available if the looper desires to use it in the future. 
In fact, I firmly believe that the remote overrides should have used an extra level of authentication when it was first implemented via an OTP. Loopers that are already using the feature have come to expect no extra OTP, though. But, this code is meant to move into a more secure direction wrt doing things remotely.

Reply:
Here is a long thread about the feature: https://loop.zulipchat.com/#narrow/stream/144182-development/topic/Remote.20bolus.20.2F.20remote.20carbs   
On my March 30 posting I talk about OTP for overrides too...

Reply:
I don't agree that this should be changed. Having the ability to enable more security is highly desirable.